pkg/inject/virtualgateway_envoy.go

```diff
@@ -15,6 +15,8 @@ const envoyVirtualGatewayEnvMap = `
   "APPMESH_VIRTUAL_NODE_NAME": "mesh/{{ .MeshName }}/virtualGateway/{{ .VirtualGatewayName }}",
   "APPMESH_PREVIEW": "{{ .Preview }}",
   "ENVOY_LOG_LEVEL": "{{ .LogLevel }}",
+  "ENVOY_ADMIN_ACCESS_PORT": "{{ .AdminAccessPort }}",
+  "ENVOY_ADMIN_ACCESS_LOG_FILE": "{{ .AdminAccessLogFile }}",
```

Comment:
Need `XrayDaemonPort` here as well?

Reply:
Yes. Maybe we should include all the tracing options (jaeger/datadog) in VG as well...

Reply:
+1, consider adding those in a separate PR as that would change the scope of this change

Reply:
Included XrayDaemonPort. Will add Jaeger and Datadog support via a separate PR.